util/src/main/scala/org/ensime/util/Show.scala

```diff
@@ -0,0 +1,18 @@
+package org.ensime.util
+
+trait Show[T] {
```

Comment:
Thats why it went into utils, but sure it can do without it, I'll simplify this.